about.html

```diff
@@ -0,0 +1,216 @@
+            Electromagnetics Summit 2023 Logo        
+        </div>
+    
+        <div class="line"></div>
+    
+        <div class="content-4-mobile">
+            The logo of Electromagnetics Summit 2023 was decided on 4 January 2023.        
+        </div>
+
+        <div class="content-4-desktop">
+            The logo of Electromagnetics Summit 2023 was decided through the logo competition on 4 January 2023.
+After considering various options, the scattering of electromagnetic radiation by a single sphere,
+was chosen as representative for the development of the transition operator also known as the non-local
+conductivity of an isolated sphere. The proposed logo is provisional until a better
+element is found:
+        </div>
+        
+        
+        <div class="big-logo">
+            <img src="./images/my_logo_big.png" class="my-logo" alt="big logo">                        
+        </div>
+        
+        <div class="big-line"></div>
+    
+        <div class="content-5">            
+            See the past Electromagnetic s Summits
+        </div>
+        <div class="line"></div>
+    
+        <div class="content-6"> Take a look at mimilar past two Electromagnetics Summits which took place in Creta Greece and in Tokio Japan
+        </div>
+    
+        <div class="wrapper-past-events">
+            <div class="content-7">
+                ETOPIM 2009
+           </div>
+           
+           <div class="content-8">
+               PIERS 2018
+           </div>           
+        
+        </div>
+        
+    
+        
+    </div>
+
+    <div class="sponsors-mobile">
+        <div class="sponsor-1"> Partner</div>
+        <div class="line"></div>
+        <div class="container-items">
+            <div > <img src="./images/sponsor-2.png" alt="Mozilla"></div>
+            <div> <img src="./images/sponsor-3.png" alt="Google"></div>
+            <div> <img src="./images/sponsor-4.png" alt="Naver"></div>
+        </div>
+
+        <div class="container-items-2">
+            <div> <img src="./images/sponsor-5.png" alt="daumkakao"></div>
+            <div> <img src="./images/sponsor-6.png" alt="airbnb"></div>            
+        </div>        
+    </div>
+
+    <div class="container-items-cc-mobile" >
+        <div class="cont-row">
+            <span class="material-icons" >
+                copyright
+            </span>
+        </div>
+
+        <div>Creative commons <br> 
+            Korea </div>
+        <div class="ccright"> 2015 Creative Commons Korea. 
+            <br>Some Rights Reserved. </div>
+    </div>      
+
+    <div class="container-items-cc-desktop-about" >
+        <div class="cont-row">
+            <span class="material-icons" >
+                copyright
+            </span>
+        </div>
+
+        <div style="font-weight: 900;">Creative commons <br> 
+            Korea </div>
+        <div class="ccright" style="font-size: small;"> 2015 Creative Commons Korea. 
```

Comment:
- [ ] As the previous code reviewer mentioned, kindly don't use the inline style for reusability and readability.